client/my-sites/domains/domain-management/edit/domain-types/site-redirect-type.jsx

```diff
@@ -0,0 +1,203 @@
+				siteDomain={ selectedSite.domain }
+				purchase={ purchase }
+				compact={ true }
+				withTextStatus={ true }
+				toggleSource="site-redirect-status"
+			/>
+		);
+
+		return content && <WrapDomainStatusButtons>{ content }</WrapDomainStatusButtons>;
+	}
+
+	renderAutoRenew() {
+		const { isLoadingPurchase } = this.props;
+
+		if ( isLoadingPurchase ) {
+			return (
+				<WrapDomainStatusButtons className="domain-types__auto-renew-placeholder">
+					<p />
+				</WrapDomainStatusButtons>
+			);
+		}
+
+		return this.renderAutoRenewToggle();
+	}
+
+	planUpsellForNonPrimaryDomain() {
+		const { domain } = this.props;
+
+		return (
+			<NonPrimaryDomainPlanUpsell
+				tracksImpressionName="calypso_non_primary_domain_settings_plan_upsell_impression"
+				tracksClickName="calypso_non_primary_domain_settings_plan_upsell_click"
+				domain={ domain }
+			/>
+		);
+	}
+
+	handlePaymentSettingsClick = () => {
+		this.props.recordPaymentSettingsClick( this.props.domain );
+	};
+
+	render() {
+		const { domain, selectedSite, purchase, isLoadingPurchase, moment } = this.props;
+		const { name: domain_name } = domain;
+
+		const { statusText, statusClass, icon } = this.resolveStatus();
+
+		const newStatusDesignAutoRenew = config.isEnabled( 'domains/new-status-design/auto-renew' );
+		const newDomainManagementNavigation = config.isEnabled(
+			'domains/new-status-design/new-options'
+		);
+
+		return (
+			<div className="domain-types__container">
+				{ selectedSite.ID && ! purchase && <QuerySitePurchases siteId={ selectedSite.ID } /> }
+				{ this.planUpsellForNonPrimaryDomain() }
+				<DomainStatus
+					header={ domain_name }
+					statusText={ statusText }
+					statusClass={ statusClass }
+					icon={ icon }
+				>
+					<ExpiringCreditCard
+						selectedSite={ selectedSite }
+						purchase={ purchase }
+						domain={ domain }
+					/>
+				</DomainStatus>
+				<Card compact={ true } className="domain-types__expiration-row">
+					<div>
+						{ this.props.translate( 'Expires: %(expiry_date)s', {
+							args: {
+								expiry_date: moment( domain.expiry ).format( 'LL' ),
+							},
+						} ) }
+					</div>
+					{ this.renderDefaultRenewButton() }
+					{ ! newStatusDesignAutoRenew && domain.currentUserCanManage && (
+						<WrapDomainStatusButtons>
+							<SubscriptionSettings
+								type={ domain.type }
+								compact={ true }
+								subscriptionId={ domain.subscriptionId }
+								siteSlug={ this.props.selectedSite.slug }
+								onClick={ this.handlePaymentSettingsClick }
+							/>
+						</WrapDomainStatusButtons>
+					) }
+					{ newStatusDesignAutoRenew && domain.currentUserCanManage && this.renderAutoRenew() }
+				</Card>
+				{ newDomainManagementNavigation ? (
+					<DomainManagementNavigationEnhanced
+						domain={ domain }
+						selectedSite={ this.props.selectedSite }
+						purchase={ purchase }
+						isLoadingPurchase={ isLoadingPurchase }
+					/>
+				) : (
+					<DomainManagementNavigation
```

Comment:
We'll soon clean up the old navigation component (`DomainManagementNavigation`) so no need to add it here at all. Just always render `DomainManagementNavigationEnhanced`